### PR TITLE
mm/vm: minor cleanups

### PR DESCRIPTION
**Pull request by 00xc**
Minor cleanup and readability changes.

### CODE REVIEW

File: src/mm/vm/mapping/file_mapping.rs [hunk 1]
```diff
@@ -219,13 +221,14 @@ impl VirtualMapping for VMFileMapping {
...
         let page_size = self.page_size();
+        let page_size_bytes = usize::from(page_size);
         if write {
             if let Some(write_copy) = self.write_copy.as_mut() {
                 // This is a writeable region with copy-on-write access. The
                 // page fault will have occurred because the page has not yet
                 // been allocated. Allocate a page and copy the readonly source
                 // page into the new writeable page.
-                let offset_aligned = offset & !(page_size - 1);
+                let offset_aligned = offset & !(page_size_bytes - 1);
```
Comment (stefano-garzarella) [inline src/mm/vm/mapping/file_mapping.rs]: What about adding `PageSize::mask()`?
I mean something like this:
```diff
diff --git a/src/mm/vm/mapping/file_mapping.rs b/src/mm/vm/mapping/file_mapping.rs
index 51540aa..a687442 100644
--- a/src/mm/vm/mapping/file_mapping.rs
+++ b/src/mm/vm/mapping/file_mapping.rs
@@ -222,7 +222,6 @@ impl VirtualMapping for VMFileMapping {
         write: bool,
     ) -> Result<VMPageFaultResolution, SvsmError> {
         let page_size = self.page_size();
-        let page_size_bytes = usize::from(page_size);
 
         if !write {
             return Err(SvsmError::Mem);
@@ -236,7 +235,7 @@ impl VirtualMapping for VMFileMapping {
         // page fault will have occurred because the page has not yet
         // been allocated. Allocate a page and copy the readonly source
         // page into the new writeable page.
-        let offset_aligned = offset & !(page_size_bytes - 1);
+        let offset_aligned = offset & page_size.mask();
         write_copy.get_alloc_mut().alloc_page(offset_aligned)?;
         let paddr_new_page = write_copy.map(offset_aligned).ok_or(SvsmError::Mem)?;
         copy_page(vmr, &self.file, offset_aligned, paddr_new_page, page_size)?;
diff --git a/src/types.rs b/src/types.rs
index 770b213..f805cb9 100644
--- a/src/types.rs
+++ b/src/types.rs
@@ -17,6 +17,12 @@ pub enum PageSize {
     Huge,
 }
 
+impl PageSize {
+    pub fn mask(self) -> usize {
+        !(usize::from(self) - 1)
+    }
+}
+
 impl From<PageSize> for usize {
     fn from(psize: PageSize) -> Self {
         match psize {
```

We can do also in another PR if you prefer to keep this simple.

Comment (00xc) [reply src/mm/vm/mapping/file_mapping.rs]: Probably best left for a separate PR :)
